src/librustdoc/html/render/mod.rs

```diff
@@ -1328,7 +1328,7 @@ fn render_impl(
                         .map(|item| format!("{}.{}", item.type_(), name));
                     write!(
                         w,
-                        "<h4 id=\"{}\" class=\"{}{}{}\">",
+                        "<div id=\"{}\" class=\"{}{}{}\">",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role ?

Reply:
So it should be:

```rust
                        "<div id=\"{}\" class=\"{}{}{}\" role="heading" aria-level="4">",
```

if I understood correctly?

Reply:
Oh, if so, that'll also allow me to simplify the CSS changes quite a lot.

Reply:
Yeah, that's what I'm thinking.